tools/run_tests/run_xds_tests.py

```diff
@@ -1574,6 +1575,7 @@ def test_timeout(gcp, original_backend_service, instance_group):
                                     testcase_name, rpc, status, qty, want)
                         success = False
                 if success:
+                    logger.info('success')
```

Comment:
optional: Maybe we want something more descriptive here? Same for another 'success' below.

Reply:
This seems to be the convention for now: 5 other test just print a simple 'success' when they are done.  Technically it's not necessary (all the other tests don't print anything at all when they pass), and IMO, something like this belongs in the main test loop instead.